src/main/java/org/sunbird/ratings/model/UpdatedValues.java

```diff
@@ -0,0 +1,32 @@
+package org.sunbird.ratings.model;
+
+public class UpdatedValues {
```

Comment:
Is there a neat way to include this in the Rating bean class itself? Having a separate POJO does not seem like a good idea

Reply:
Done